libs/community/langchain_community/chat_models/reka.py

```diff
@@ -339,11 +331,9 @@ async def _agenerate(
 
     def get_num_tokens(self, text: str) -> int:
         """Calculate number of tokens."""
-        if self.count_tokens is None:
-            raise NotImplementedError(
-                "get_num_tokens() is not implemented for Reka models."
-            )
-        return self.count_tokens(text)
+        if self._tiktoken_encoder is None:
```

Comment:
suggest making this into a more generic `token_counter` attribute that's not private